.travis.yml

```diff
@@ -23,6 +23,9 @@ before_install:
   - $SHELL --version 2> /dev/null || dpkg -s $SHELL 2> /dev/null || which $SHELL
   - curl --version
   - wget --version
+  - bash --version | head
+  - zsh  --version
+  - dpkg -s dash | grep ^Version | awk '{print $2}'
```

Comment:
Oh yes, forgot to delete it, I wonder if we could directly print the versions instead of dynamic call the shell? Just like we print both `wget` and `curl` versions?